src/pages/dataTableUtilities/actions/cellActions.js

```diff
@@ -26,6 +26,11 @@ import { selectPageState } from '../../../selectors/pages';
  *
  * @param {String} rowKey Key of the row to edit.
  */
+export const focusCell = (rowKey, columnKey) => ({
+  type: ACTIONS.FOCUS_CELL,
+  payload: { rowKey, columnKey },
```

Comment:
I think you should only need the row key here as we would only need to know which row is being focused to highlight it. We also need the route for these actions i.e. like `refreshRow`

```
export const focusCell = (rowKey, route) => ({
  type: ACTIONS.FOCUS_CELL,
  payload: { rowKey, route },
```

Also the constant isn't defined